src/main/scala/codecheck/github/models/AbstractJson.scala

```diff
@@ -59,4 +60,16 @@ class AbstractJson(value: JValue) {
   }
 
   override def toString = JsonMethods.pretty(value)
+
+  def seqOpt[T](path: String): Seq[T] = {
+    path.split("\\.").foldLeft(value) { (v, s) =>
+      v \ s
+    } match {
+      case JNothing => Nil
+      case JNull => Nil
+      case v: JArray => v.values.map(_.asInstanceOf[T])
+    }
+  }
+
+  def seq(path: String): Seq[String] = seqOpt(path)
```

Comment:
About seq and seqOpt, I think there might be better implementation.
We'll consider later.

You don't have to fix it in this branch.